src/Agent/CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ Fixes Issue [#XX](https://github.com/newrelic/newrelic-dotnet-agent/issues/XX)
 
 ### Fixes
 * Fixes [#223](https://github.com/newrelic/newrelic-dotnet-agent/issues/223) so the agent can be compatible with ASP.NET Core 5 RC1.
-
+* Fixes `HttpClient` instrumentation for .NET 5 RC1. See [#227](https://github.com/newrelic/newrelic-dotnet-agent/issues/227).
```

Comment:
How about:

"Fixes issue in .NET 5 applications where external calls made with `HttpClient` may not get instrumented. For example, calls made with `HttpClient.GetStringAsync` would be missed. ([#235](https://github.com/newrelic/newrelic-dotnet-agent/pull/235))"

I like the practice of noting the PR that fixed the bug rather than the issue in the release notes. The PR itself should reference the issue.